lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -46,7 +46,7 @@
 import matplotlib.axes as maxes
 from matplotlib.path import Path
 from mpl_toolkits.axes_grid1 import mpl_axes
-from .axisline_style import AxislineStyle
+from .axisline_style import AxislineStyle  # noqa
```

Comment:
Maybe one should deprecate this? Not sure if someone is actually importing it from here?